Examples/ServiceNow/Create_SQLite_Settings_DB.ps1

```diff
@@ -0,0 +1,73 @@
+﻿<#
+    .SYNOPSIS
+    Store server details and credentials in a data table for use in
+	importing devices From ServiceNow into a Dashworks Device Feed.
+
+    .DESCRIPTION
+    Creates a database of the name given in the same path as the script.
+	Stores the gioven credentials in a datatable created for that purpose.
+#>
+
+$DLLPath = "$PSScriptRoot\System.Data.SQLite.dll"
```

Comment:
I cant add this comment to the dll itself as gtihub doesn't allow it! 

Are we allowed to distribute the dll? whats the license? 

Would it be better to include a link to the source download instead? So that the latest version is used and we aren't distributing it ourselves?

Reply:
https://www.sqlite.org/copyright.html

SQLite is public domain.

https://system.data.sqlite.org/index.html/doc/trunk/www/index.wiki

The dll comes from here and is unlimited in its use or distribution.

The comment section in the main example contains the text:

    Uses the supplied .\System.Data.SQLite.dll which requires .Net 4.6. This can be changed to suit.
    Alternate versions can be found at https://system.data.sqlite.org/